packages/core/src/llm/model/router.ts

```
@@ -15,20 +15,18 @@ import type {
   GatewayLanguageModelWithStreamTransport,
   GatewayStreamTransportHandle,
   MastraModelGatewayInterface,
-  GatewayAuthRequest,
 } from './gateways/base.js';
-import { findGatewayForModel, getGatewayId } from './gateways/index.js';
+import { GatewayManager, getGatewayId } from './gateways/index.js';
+import { defaultGateways } from './gateways/defaults.js';
```

ESLint

[error] `./gateways/defaults.js` import should occur before import of `./gateways/index.js`

(import/order)